manim/mobject/coordinate_systems.py

```diff
@@ -1355,6 +1355,25 @@ class NumberPlane(Axes):
 
     .. note:: If :attr:`x_length` or :attr:`y_length` are not defined, the plane automatically adjusts its lengths based
         on the :attr:`x_range` and :attr:`y_range` values to set the unit_size to 1.
+
+    Examples
+    --------
+
+    .. manim:: NumberPlaneExample
+        :save_last_frame:
+
+        class NumberPlaneExample(Scene):
+            def construct(self):
+                number_plane = NumberPlane(
+                    x_range=[-10, 10, 1],
+                    y_range=[-10, 10, 1],
+                    background_line_style={
+                        "stroke_color": RED_D,
```

Comment:
It might be nice to see a different colour since I think red is probably a little over-represented in the docs right now. Maybe `PURPLE`, `PINK` or `ORANGE`?

Reply:
Changed the color to TEAL. @hydrobeam